HunnitLog/HunnitLog/View/Home/AchievedGoalRow.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AchievedGoalRow.swift
+//  HunnitLog
+//
+//  Created by 초이 on 2020/11/26.
+//
+
+import SwiftUI
+
+struct AchievedGoalRow: View {
```

Comment:
제가 remote 를 모두 추가하질 않아서 바로 로컬에 코드 땡겨서 보진 못했는데요, 
이런식으로 constant 값들을 private enum 으로 선언하고, 실제 body 내부에는 상수값을 별도로 사용하지 않도록 정리하는건 어떨까요?? 
지금은 뭐 별 상관없겠지만, 동일한 값들끼리 묶여있다면 추후 수정이 필요할 때, 요부분만 확인, 수정하면 되서 편리할 것 같습니다. 

그리고 struct 가 아닌 enum 으로 제안드린 이유는 struct 로 생성하면 외부에서 init() 에 접근이 가능하지만, enum 은 외부에서 init() 접근이 불가능 하여, "값 정의" 를 위해 사용된 부분으로써의 의미를 지키며 사용상의 실수를 줄일 수 있기 때문에 enum 으로 제안드려봅니다. (struct 도 private init() 을 선언해두면 되긴 하지만,,, 의미없는 코드가 늘어나는것이라 enum 으로 제안드려요)

```suggestion
struct AchievedGoalRow: View {
    private enum Constants {
        static let font: Font = .system(size: 22)
        static let circleSize: Size = ...
    }
```

Reply:
현재 dev 에있는 `KeyWordButton.swift` 파일에서 사용되는` private enum Constants` 와 같은 형태가 어떨까 하고 제안드려봅니다

Reply:
안그래도 디자이너님께 혹시 폰트나 크기 등에 규칙들이 있는지,
만약 있다면, 정리해주시면 아예 템플릿처럼 만들까 해서 여쭤보았지만
나름의 규칙이 있긴한데 명확한 규칙이아니라고 하셔서... 추후에 정리가 되거나 하면 저렇게 묶는것도 좋을 것 같아요~

Reply:
넹~!